### ISSUE TITLE
[FR] install checker

### ISSUE THREAD

**Issue by AlexanderGoerlich** [senseexcel/ser-web-management#82]
Show everything that has been checked and not only what is wrong right now

**Comment by thomashaenig** [comment]
into Info section, make new subpoint status

REST ENGINE - check
- make health check (endpoint: healthstatus)

QLIK - check
- test connection to qlik (??? open session)

SER TAGS
- check if ser tags exists and how many on apps, tasks, connections, reports

**Comment by r-hannuschka** [comment]
blocked by #225 

i merged #225 into this branch since we allready implement websocket ping in enigma.service.ts that i needed to check qlik websocket connection is alive.

**Comment by thomashaenig** [comment]
duplicate of #54
